rules/linux/execution_netcon_from_rwx_mem_region_binary.toml

```diff
@@ -0,0 +1,93 @@
+[metadata]
+creation_date = "2024/03/13"
+integration = ["auditd_manager", "endpoint"]
+maturity = "production"
+min_stack_comments = "The sampling feature within EQL was introduced in 8.6.0"
+min_stack_version = "8.6.0"
+updated_date = "2024/03/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Monitors for the execution of a unix binary with read, write and execute memory region permissions, followed by a 
+network connection. The mprotect() system call is used to change the access protections on a region of memory that has
+already been allocated. This syscall allows a process to modify the permissions of pages in its virtual address space,
+enabling or disabling permissions such as read, write, and execute for those pages. RWX permissions on memory is in many
+cases overly permissive, and should (especially in conjunction with an outbound network connection) be analyzed 
+thoroughly. 
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "auditbeat-*", "logs-auditd_manager.auditd-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Network Connection from Binary with RWX Memory Region"
+references = [
+    "https://man7.org/linux/man-pages/man2/mprotect.2.html"
+]
+risk_score = 47
+rule_id = "32300431-c2d5-432d-8ec8-0e03f9924756"
+setup = """## Setup
```

Comment:
Not sure if you have done so already but I might test the markdown render by running `view-rule`, flatten to NDJSON and import rule.